data/sql_updates/create_omnibus_dates.sql

```diff
@@ -21,8 +21,17 @@ returns text as $$
     end
 $$ language plpgsql;
 
-
-
+create or replace function get_report_category(report_type text)
+returns text as $$
+    begin
+        return case
+            when report_type in ('Q1', 'Q2', 'Q3') then 'Q'
```

Comment:
If we want to group things in the API, wouldn't we want to add the year end for quarterly filings and monthly filings too?  Or would the webapp ask for monthly and year end deadlines? That seems reasonable too.